README.md

```diff
@@ -162,14 +169,14 @@ test('uploading an image', async function (assert) {
 If the file isn't uploaded to the server, you don't need to use the mirage helper. The same approach applies to all types of files; encode them as a Base64 data url or read them from a file as a blob.
 
 ```javascript
-import upload from '../helpers/upload';
+/* global upload */
 
 moduleForAcceptance('/notes');
 
 test('showing a note', async function (assert) {
   let file = File.fromDataUrl('data:text/plain;base64,SSBjYW4gZmVlbCB0aGUgbW9uZXkgbGVhdmluZyBteSBib2R5');
 
-  await upload('#upload-note', file, 'douglas_coupland.txt');
+  await upload('#upload-note input[type="file"]', file, 'douglas_coupland.txt');
```

Comment:
Other examples with templates use `id` selectors (like `"#upload-image"`) to reference elements that aren't the `<input>`

Since these examples don't have corresponding example templates, I've added the `input[type="file"]` selector here to indicate that the selector we want is for the file upload _input_.

Reply:
That works! I think I chose not to so we didn't expose the private internals of the `{{file-upload}}` template.